File: component/component-lovelycat-httpapi/src/test/java/love/forte/test/Test2.java
/*
 *
 *  * Copyright (c) 2020. ForteScarlet All rights reserved.
 *  * Project  simple-robot
 *  * File     MiraiAvatar.kt
 *  *
 *  * You can contact the author through the following channels:
 *  * github https://github.com/ForteScarlet
 *  * gitee  https://gitee.com/ForteScarlet
 *  * email  ForteScarlet@163.com
 *  * QQ     1149159218
 *
 */

package love.forte.test;

import love.forte.simbot.api.message.containers.GroupInfo;
import love.forte.simbot.component.lovelycat.message.event.BaseLovelyCatEvents;


/**
 * @author ForteScarlet
 */
public class Test2 {
    public static void main(String[] args) throws InterruptedException {
        GroupInfo groupInfo = BaseLovelyCatEvents.lovelyCatGroupInfo("18367333210@chatroom", "xxx群聊");
        System.out.println(groupInfo.getGroupCodeNumber());

    }
}

File: http/client-restTemplate/src/test/java/Test2.kt
/*
 *
 *  * Copyright (c) 2020. ForteScarlet All rights reserved.
 *  * Project  component-onebot
 *  * File     Test2.kt
 *  *
 *  * You can contact the author through the following channels:
 *  * github https://github.com/ForteScarlet
 *  * gitee  https://gitee.com/ForteScarlet
 *  * email  ForteScarlet@163.com
 *  * QQ     1149159218
 *  *
 *  *
 *
 */



fun main() {


    val wait = waitFor<Int> {
        println("n1")
        val n1 = wait()
        println("n1: $n1")
        println("n2")
        val n2 = wait()
        println("n2: $n2")
        println("n1 n2 finished.")
        println("total: ${n1 + n2}")
    }



    println("bef wake1")
    wait.wakeup(1)
    println("bef wake2")
    wait.wakeup(4)


    println("end?")


}
